pom.xml

```diff
@@ -29,6 +29,9 @@
         <db.username>root</db.username>
         <db.password>5472</db.password>
         <spring.profiles.active>default</spring.profiles.active>
+
+        <logging.file>${project.name}.log</logging.file>
```

Comment:
I think we need to have logging.file only as a ${project.name}. Why? See next comments for logbackx.groovy